sebs/local/local.py

```
@@ -68,298 +83,410 @@ def __init__(
+                )
+                return func_obj
             except requests.exceptions.ConnectionError:
-                time.sleep(0.25)
-                attempts += 1
+                if attempt < max_retries - 1:
+                    time.sleep(0.25)
+                else:
+                    raise RuntimeError(
+                        f"Couldn't start {func_name} in container {running_container.id} "
+                        f"(URL: {func_obj.url}). Server did not become alive."
+                    )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)